lib/Common/PlatformAgnostic/DateTimeInternal.h

```diff
@@ -61,7 +61,7 @@ namespace DateTime
         uint32 lastTimeZoneUpdateTickCount;
 
         void UpdateTimeZoneInfo();
-        UtilityPlatformData(): lastTimeZoneUpdateTickCount(0) { }
+        UtilityPlatformData() : lastTimeZoneUpdateTickCount(0) { GetTimeZoneInformation(&timeZoneInfo); }
```

Comment:
Instead of directly calling `GetTimeZoneInformation` here, would it be acceptable to call `UpdateTimeZoneInfo` to do it instead?